### PR TITLE
fix: inline mdx images

### PR DESCRIPTION
**Pull request by kellyjosephprice**
[![PR App][icn]][demo] |
:-------------------:|

## 🧰 Changes

Fixes loading inline images.

The editor was throwing away inlined MDX images! This PR converts them to plain inline markdown images.

## 🧬 QA & Testing

- [Broken on production][prod].
- [Working in this PR app][demo].

[demo]: https://markdown-pr-PR_NUMBER.herokuapp.com
[prod]: https://SUBDOMAIN.readme.io
[icn]: https://user-images.githubusercontent.com/886627/160426047-1bee9488-305a-4145-bb2b-09d8b757d38a.svg

### CODE REVIEW

File: __tests__/lib/mdast/index.test.ts [hunk 1]
```diff
@@ -1,21 +1,43 @@
...
+import tablesJson from './tables/out.json';
+
+import variablesMdx from './variables/in.mdx?raw';
+import variablesJson from './variables/out.json';
+
+import inlineImagesMdx from './images/inline/in.mdx?raw';
+import inlineImagesJson from './images/inline/out.json';
+
+import inlineImageAttributesMdx from './images/inline-attributes/in.mdx?raw';
+import inlineImageAttributesJson from './images/inline-attributes/out.json';
```
Comment (kellyjosephprice) [inline __tests__/lib/mdast/index.test.ts:16]: This is just a nicer experience, you got automatic re-runs, easier to add `.only`.

**Comment by rafegoldberg** [comment]
Won't converting ‘em to raw `` image syntax loose details like the `[border]` or `[align]` properties?

**Comment by kellyjosephprice** [comment]
> So this will convert any inline image to the raw `` image syntax? Won't that loose details like the `[border]` or `[align]` properties?

Hmm, I guess I was trying to maintain parity with existing behavior. But MDX makes it a lot easier to support inline images. Yea, ignore this, I'll fix this in the editor.

**Comment by kellyjosephprice** [comment]
Kept the tests around though!

**Comment by rafegoldberg** [comment]
## This PR was released!

🚀 Changes included in [<b>v7.6.5</b>](/readmeio/markdown/releases/tag/v7.6.5)
<br>

**Pull request merged**